### ISSUE TITLE
[ENH] Implement LOCI

### ISSUE THREAD

**Issue by jpaillard** [mind-inria/hidimstat#678]
### Please confirm that you have searched existing issues in the repository.

- [x] Yes

### Please describe the feature that you would like to see improved or added.

Add an implementation of Leave One Covariate In. 

### Describe the solution you would like to see implemented.

The idea is to have an estimator that computes the response $Y$ for any model from the individual features $X^j$. 
Then compare the loss of that univariate model to a "null model," which would be something like a constant: the mean in regression and class prevalence for classification.

**Comment by bthirion** [comment]
Yes, we need that one too.

**Comment by GrituX** [comment]
I think it's good in term of library completedness to have it, but I'm wondering, do people use this method ?

**Comment by jpaillard** [comment]
I see it as an extension of the [f_regression](https://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.f_regression.html#f-regression) (or classification) from sklearn to non-linear models. 
It could provide complementary information to LOCO and CFI. For instance, for highly correlated features that are also highly predictive, they would get a very low LOCO importance but high LOCI

**Comment by GrituX** [comment]
True, I did not consider that.
Do you want to work on the implementation ? If not, I can do that next week.

**Comment by jpaillard** [comment]
As you want. If you have the time, go ahead
